src/libs/queries/factories/transaction.queries.ts

```diff
@@ -0,0 +1,9 @@
+import { createQueryKeys } from '@lukemorales/query-key-factory'
+import * as TransactionApi from '../../apis/Transaction.api.ts'
+
+export default createQueryKeys('transactions', {
+	transactionList: () => ({
```

Comment:
Make this query reusable (in the aside to count how many transactions there is and in the transaction listing component ).

<img width="312" alt="image" src="https://github.com/user-attachments/assets/b86d38dc-36ca-43d5-a574-2fc436706e64" />